src/keyboard.c

```diff
@@ -132,7 +140,6 @@ handle_compositor_keybindings(struct wl_listener *listener,
 					desktop_cycle_view(server, server->cycle_view,
 						backwards ? LAB_CYCLE_DIR_BACKWARD : LAB_CYCLE_DIR_FORWARD);
 				osd_update(server);
-				damage_all_outputs(server);
```

Comment:
Why is this call removed? Is there a reason it's actually not necessary, or was it causing misbehavior?

Reply:
From what testing I've done, it looks like it isn't necessary.

Reply:
It appeared to be unnecessary since `damage_all_outputs()` was already called on line 120.
Thanks @ARDiDo for confirming!